src/wp-admin/includes/file.php

```diff
@@ -1746,7 +1749,23 @@ function _unzip_file_ziparchive( $file, $to, $needed_dirs = array() ) {
 			return new WP_Error( 'mkdir_failed_ziparchive', __( 'Could not create directory.' ), $_dir );
 		}
 	}
-	unset( $needed_dirs );
+
+	/**
+	 * Filters archive unzipping to override with a custom process.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param null|true|WP_Error $result         The result of the override. True on success, otherwise WP Error. Defautl null.
```

Comment:
Fix typo, default

Reply:
Fixed in [b20c999](https://github.com/WordPress/wordpress-develop/pull/4220/commits/b20c9994e4ca9aa652297fd186dfb32419b8a6ae). Thanks Andy!